metrics_sdk/lib/opentelemetry/sdk/metrics/meter_provider.rb

```diff
@@ -116,6 +116,15 @@ def add_metric_reader(metric_reader)
           end
         end
 
+        # @api private
+        def register_synchronous_instrument(instrument)
+          @mutex.synchronize do
+            @metric_readers.each do |mr|
+              instrument.register_with_new_metric_store(mr.metric_store)
```

Comment:
How do we envisage setting a non-default aggregation?

Reply:
There are a few ways aggregation could be configured, some examples via view here.

>[Note: Implementors MAY choose the best idiomatic practice for their language to represent the semantic of an Aggregation and optional configuration parameters.](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk.md#aggregation)

```
# Use Histogram with custom boundaries
meter_provider
  .add_view(
    "X",
    aggregation="ExplicitBucketHistogram",
    aggregation_params={"Boundaries": [0, 10, 100]}
    )
```

```
// Use Histogram with custom boundaries
meterProviderBuilder
  .AddView(
    instrumentName: "X",
    aggregation: new ExplicitBucketHistogramAggregation(
      boundaries: new double[] { 0.0, 10.0, 100.0 }
    )
  );
```

I think in the trace implementation we have more examples of the latter where we allow consumers to instantiate a configurable something and pass it into the respective attribute writer. 

Something not totally clear to me is whether the following line could be interpreted to mean that the metric reader can also configure the aggregation. That doesn't seem to follow prior wording in the SDK spec. 

>[The default output aggregation (optional), a function of instrument kind. If not configured, the [default aggregation](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk.md#default-aggregation) SHOULD be used.](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk.md#metricreader)

Reply:
Discussed IRL: aggregations can be configured either in the `MetricReader` or in `View`s. We'll be tackling those in follow-up PRs, in that order, so we'll handle the wiring at that time.